lingua_franca/format.py

```diff
@@ -290,6 +293,9 @@ def pronounce_number(number, lang='', places=2, short_scale=True,
 
     Args:
         number: the number to pronounce
+        lang (str, optional): an optional BCP-47 language code, if omitted
+                              the default language will be used.
+        places (int): number of decimal places to express number to, default 2
```

Comment:
"number of decimal places to express" might be better. idk, i can't come up with good phrasing, except that the trailing 'to' is giving me fits

Reply:
Good suggestion - fixed.